compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -550,10 +561,10 @@ impl<'a> Parser<'a> {
             token::Not => make_it!(this, attrs, |this, _| this.parse_unary_expr(lo, UnOp::Not)),
             // `~expr`
             token::Tilde => make_it!(this, attrs, |this, _| this.recover_tilde_expr(lo)),
-            // `-expr`
-            token::BinOp(token::Minus) => {
-                make_it!(this, attrs, |this, _| this.parse_unary_expr(lo, UnOp::Neg))
-            }
+            // // `-expr`
+            // token::BinOp(token::Minus) => {
+            //     make_it!(this, attrs, |this, _| this.parse_unary_expr(lo, UnOp::Neg))
+            // }
```

Comment:
Did you mean to remove this? (given the comment below, I don't think this should be touched at all...)